source/_integrations/default_config.markdown

```diff
@@ -11,7 +11,7 @@ This integration is a meta-component and configures a default set of integration
 
 - [automation](/integrations/automation/)
 - [cloud](/integrations/cloud/)
-- [config](/integrations/config/)
+- [configuration](/integrations/config/)
```

Comment:
Is this going well?

Reply:
Yes, the parent PR's on the upstream repos take care of the "visual" naming versus the domain used.
The merge of this can happen already, without any impact.

Reply:
The change won't break anything, but the integration is named `config`.
This should be reverted.

Reply:
No, I don't agree, this is end-user documentation. We should not talk about DOMAIN keys (since that is what it is), but about actual names.

To correct the name in the upstream, https://github.com/home-assistant/home-assistant/pull/31782 is pending.

Reply:
ok

Reply:
You made me think, the links where domain keys, so I've just pushed a change on this PR, to make the link text of that list to contain the actual integration names, based on the component manifest files.

Reply:
I like that change.